staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -38,7 +38,9 @@ type HorizontalPodAutoscalerSpec struct {
 	// reference to scaled resource; horizontal pod autoscaler will learn the current resource consumption
 	// and will set the desired number of pods by using its Scale subresource.
 	ScaleTargetRef CrossVersionObjectReference `json:"scaleTargetRef" protobuf:"bytes,1,opt,name=scaleTargetRef"`
-	// lower limit for the number of pods that can be set by the autoscaler, default 1.
+	// MinReplicas is the lower limit for the number of replicas to which the autoscaler can scale down.
+	// It defaults to 1 pod. MinReplicas is allowed to be 0 if at least one Object or External metric is configured.
```

Comment:
nit: use `minReplicas` to match the serialized name API consumers will see.

include a mention that setting minReplicas=0 requires the alpha feature gate HPAScaleToZero to be enabled.

Reply:
Updated comments in all API versions.